core/container.go

```diff
@@ -48,6 +48,9 @@ const (
 	QodanaOutOfMemoryExitCode = 137
 	// QodanaEapLicenseExpiredExitCode reports an expired license.
 	QodanaEapLicenseExpiredExitCode = 7
+	// QodanaTimeoutExitCodePlaceholder is not a real exit code (it is not obtained from IDE process!)
+	// Placeholder used to identify the case when the analysis reached timeout
+	QodanaTimeoutExitCodePlaceholder = 1000
```

Comment:
I think most shells don't expect exit codes >255 - if for some reason this code ever "escapes" the program, it would be nice if it was a "normal" code.

Reply:
This code is never returned from CLI, it's used internally to mark the case when the timeout happens, I tried to represent this idea in "Placeholder" suffix and comment, but I guess it doesn't work